boa_engine/src/vm/call_frame/env_stack.rs

```diff
@@ -3,7 +3,11 @@
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub(crate) enum EnvEntryKind {
     Global,
-    Loop,
+    Loop {
+        /// This is used to keep track of how many iterations a loop has done.
+        #[cfg(feature = "runtime-limits")]
+        iteration_count: u64,
```

Comment:
Just curious on what your thoughts might be about setting this as a `u16` vs. `u64`.

Reply:
I set it because I felt like that was too restrictive `[0, 65535]`for `u16`, `u32` with `[0, 4294967295]` is better but it is possible to iterate over it (takes like a second on node with jitting), but `u64` should be more than enough for everyone `[0, 18446744073709551615]` :D

Reply:
😆 I'd agree that u64 is more than enough. 

I mostly brought it up because of how large u64 is. It may be worth keeping track or maybe getting a poll/input to see if people are setting runtime limits past a certain point and see if we should lower to a `u32` or something. But that could just be me being overly concerned and thinking about it completely wrong.